util/regtool.py

```diff
@@ -11,10 +11,10 @@
 import sys
 from pathlib import Path
 
-from reggen import (gen_cheader, gen_dv, gen_fpv, gen_html,
-                    gen_json, gen_rtl, gen_rust, gen_selfdoc, version)
-from reggen.ip_block import IpBlock
+from reggen import (gen_cheader, gen_dv, gen_fpv, gen_html, gen_json, gen_rtl,
+                    gen_rust, gen_sec_cm_testplan, gen_selfdoc, version)
 from reggen.countermeasure import CounterMeasure
```

Comment:
Funny, I thought imports were ordered alphabetically, so I don't understand this change.

Reply:
`reggen.c...` comes before `reggen.i...`. This (and few other unrelated formatting changes below) were done by `isort` and `yapf` so I did not bother questioning them :)

Reply:
My goof... it is alphabetical now :)